blog/2015/01/future-and-funding-of-science.html

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+root: ../../..
+author: Greg Wilson
+title: "The Future and Funding of Science"
+date: 2015-01-04
+time: "05:30:00"
+category: ["Opinion"]
+---
+<!-- start excerpt -->
+<p>
+  I was talking with friends over the holiday about the future of science
+  and how it might one day be funded.
+  Since it'll be ten years before I'm proven wrong,
+  it seems like a good topic with which to start the new year.
+</p>
+<!-- end excerpt -->
+<p>
+  First,
+  I believe that <em>everything</em> in the next fifty years
+  is going to be shaped by a bunch of bills coming due at the same time.
+  Thanks to our generation's selfish dithering,
+  global climate change,
+  resource depletion,
+  drug-resistant diseases,
+  the Sixth Great Extinction,
+  and aging populations in the industrialized world
+  are all going to hit us at more or less the same time.
+  Research that doesn't directly address one or more of these problems
+  will be sidelined in favor of research that does&mdash;basically,
+  trips to Mars and LHC-style "big physics" will be starved
+  so that climatology, materials science, and ecology can be fed.
+</p>
+<p>
+  Second,
+  research will be much more directed:
+  "grand challenge" initiatives modeled on the Manhattan Project and the Apollo program
+  will receive the lion's share of funding.
+  Many scientists will complain that freedom of inquiry is disappearing,
+  but this doesn't mean that fundamental research will disappear:
+  historically,
+  the Second World War was a tremendously productive period at all levels of scientific inquiry
+  precisely because scientists were working toward specific goals.
+  It does mean,
+  though,
+  that politicians (who represent the interests of the people who fund research)
```

Comment:
This is an optimistic description of politicians. (:sob:) Isn't lobbying going to continue to run this show?